### PR TITLE
Add redux store, reducer and action

### PR DESCRIPTION
**Pull request by akhror-valiev**
##Summary 
- Added redux-toolkit
- Used redux action and reducer
- Created redux store

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  reducers: {
+    addBook: (state, action) => {
+      const newBook = {
+        id: Date.now(),
+        title: action.payload.title,
+      };
+      state.push(newBook);
+    },
+  },
+
```
Comment (Gambit142) [inline src/redux/books/books.js]: - [ ] Great work creating a reducer and action creator for _**addBook.**_ However, you forgot to create an action creator for removing a book. The action creator for removing a book should remove the book from the state. Kindly implement this feature. 👍

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
```
Comment (Gambit142) [inline src/redux/categories/categories.js]: - [ ] When creating action types, it is recommended to use the [duck](https://github.com/erikras/ducks-modular-redux) pattern to name the action type. For example;
`name-of-app/reducer/action-type` `Bookstore/books/add-book`. Kindly name the action type for the category like this. 💻

File: src/redux/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/books';
+
+export default configureStore({
+  reducer: {
+    books: bookReducer,
+  },
+});
```
Comment (Gambit142) [inline src/redux/configureStore.js]: - [ ] Your store should have two types of reducers in it; one for the books and another for the category. Kindly import the category reducer and use it here. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @akhror-valiev 👋, 

Good job so far!

## Highlights
- All linters checks passed 👍 
- Installed `@reduxjs/toolkit` 👍 
- Used `configureStore` 👍 
- The structure of the files follows the `duck` pattern 👍 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @akhror-valiev      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R
- [x] Descriptive README file




 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**